Dockerfile-assemblyscript

```
@@ -0,0 +1,20 @@
+FROM node:15.14.0
+LABEL maintainer="Developer Relations Team <support@fastly.com>"
```

We tend to prefer using the open-source software distribution list for all public communication, i.e. 
```suggestion
LABEL maintainer="Fastly OSS <oss@fastly.com>"
```

---

This also applies to the Rsut image below.